cli/src/command.rs

```diff
@@ -657,18 +660,20 @@ pub fn run() -> Result<()> {
 				<E as NativeExecutionDispatch>::ExtendHostFunctions,
 			>;
 
-			use sc_service::TaskManager;
 			let registry = &runner.config().prometheus_config.as_ref().map(|cfg| &cfg.registry);
 			let task_manager = TaskManager::new(runner.config().tokio_handle.clone(), *registry)
 				.map_err(|e| Error::SubstrateService(sc_service::Error::Prometheus(e)))?;
 
+			let info_provider = timestamp_with_babe_info(6000);
```

Comment:
Would be nicer to not hardcode but get the block time value from each runtime itself.